Scripts/BuildGVFSForWindows.bat

```diff
@@ -1,4 +1,4 @@
-@ECHO OFF
+@if "%_echo%"=="" (echo off) else (echo on)
 SETLOCAL

```

Comment:
I hope you don't mind a drive-by.  I had some struggles getting these to run.  We use this technique a bit around here to allow conditionally turning echo on to help diagnose the random "Error: wheel fell off" messages you get when running batch scripts in slightly incorrect environments"

Reply:
Sounds good to me. As long as the CI builds still pass with these messages, then I'm happy with the change.